src/Temporalio/Converters/BinaryPlainConverter.cs

```diff
@@ -18,8 +18,7 @@ public class BinaryPlainConverter : IEncodingConverter
         /// <inheritdoc />
         public bool TryToPayload(object? value, out Payload? payload)
         {
-            var bytes = value as byte[];
-            if (bytes == null)
+            if (value is not byte[] bytes)
```

Comment:
I'm starting to like this language more and more as I read your PRs.

Reply:
Wait until you see F#